src/appleseed/renderer/modeling/object/meshobject.cpp

```diff
@@ -37,6 +37,7 @@
 
```

Comment:
They could make sense on curves, if you use them for feathers for instance (or grass).